src/components/Sections/Home/Container.js

```diff
@@ -106,7 +105,6 @@ const HomePageContainer = ({ location, t }) => {
 
 HomePageContainer.propTypes = {
   location: shapes.HistoryLocation.isRequired,
-  f: PropTypes.func.isRequired,
 };
```

Comment:
Please think a bit... Look in browser...

Reply:
missed t as well 🥇

Reply:
It was a typo, please return this line and just replace f with t